src/logic/messages/EventManager.cpp

```diff
@@ -105,6 +119,43 @@ void EventManager::processMessageQueue()
     if(m_EventQueue.empty())
         return;
 
+    bool print_queue = false;
```

Comment:
Btw. this is not a change request, just a remark, I think the bug existed long before, we should probably create a separate issue for that.